controllers/game.controller.js

```diff
@@ -0,0 +1,137 @@
+const connect = require("../configs/database");
+
+
+//[GET] /games
+module.exports.game = async (req, res) => {
+    try {
+        const sql = "SELECT * FROM games"
+        const conn = await connect.connection();
+        const records = await conn.execute(sql);
+
+        res.json({
+            code: 200,
+            message: "Success!",
+            data: records[0],
+        });
+    } catch (error) {
+        res.status(500).json({
+            code: 500,
+            message: "Error",
+            error: error.message
+        })
+    }
+
+}
+
+//[POST] /games
+module.exports.createGame = async (req, res) => {
+    try {
+        const conn = await connect.connection();
+        let sql = `INSERT INTO games (game_id, title, genre, price) VALUES (?, ?, ?, ?)`;
+        const newGame = req.body;
+        const values = Object.values(newGame);
+        // console.log(values);
+        
+        if(newGame.title.trim() === "" || newGame.genre.trim() === "" || isNaN(newGame.price) || newGame.price < 0){
+            res.json({
+                code: 500,
+                message: "Title or genre is invalid"
+            })
+        }
+        await conn.query(sql, values);
+
+        res.json({
+            code: 200,
+            message: "Success!",
+            data: newGame
+        })
+
+    } catch (error) {
+        res.json({
+            code: 500,
+            message: "Error!",
+            error: error.message
+        })
+    }
+
+    
+}
+
+//[GET] /games/:id
+module.exports.findGame = async (req, res) => {
```

Comment:
Change the function name to have more meaningful descriptions (e.g., findGameById)